### PR TITLE
Compose-cli is now available with Desktop stable. Remove mentions of aci-integration-beta

### PR DESCRIPTION
**Pull request by gtardif**
### Proposed changes

* Replace edge by stable in Desktop prerequisites
* Remove mentions of aci-integration-beta, compose-cli is now open source

### CODE REVIEW

**Review by usha-mandya — APPROVED** [review]
LGTM. Thanks.

File: engine/context/aci-integration.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ In addition, the integration between Docker and Microsoft developer technologies
 
 To deploy Docker containers on Azure, you must meet the following requirements:
 
-1. Download and install Docker Desktop Edge version 2.3.2.0 or later.
+1. Download and install Docker Desktop Stable version 2.3.0.5 or later, or Edge version 2.3.2.0 or later.
 
-    - [Download for Mac](https://desktop.docker.com/mac/edge/Docker.dmg){: target="_blank" class="_"}
-    - [Download for Windows](https://desktop.docker.com/win/edge/Docker%20Desktop%20Installer.exe){: target="_blank" class="_"}
+    - [Download for Mac](https://desktop.docker.com/mac/stable/Docker.dmg){: target="_blank" class="_"}
```
Comment (chris-crone) [inline engine/context/aci-integration.md:30]: I left these as edge to encourage more people to use edge. Not sure what @nebuk89 would like us to do though

**Review by chris-crone — APPROVED** [review]
LGTM

File: engine/context/aci-integration.md (hunk 1 above)
Comment (nebuk89) [reply engine/context/aci-integration.md:30]: Agree, saying it is in both but pointing to edge is good for now :) a short term issue either way

**Review by nebuk89 — APPROVED** [review]
LGTM

**Pull request merged**